From a69bdc87d7a4a7b019c4086cb007a0b48a007f4a Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 25 Aug 2022 12:36:18 -0700
Subject: [PATCH] fix: Fix build with musl when using GCC 12 (#1145)

(cherry picked from commit bb72cc26bc91bc56567dbef608088a9437f836bb)
---
 src/Config.hpp          | 2 ++
 src/core/Statistics.hpp | 1 +
 2 files changed, 3 insertions(+)

--- a/src/Config.hpp
+++ b/src/Config.hpp
@@ -25,6 +25,8 @@
 
 #include "third_party/nonstd/optional.hpp"
 
+#include <sys/types.h>
+
 #include <cstdint>
 #include <functional>
 #include <limits>
--- a/src/core/Statistics.hpp
+++ b/src/core/Statistics.hpp
@@ -21,6 +21,7 @@
 #include <core/StatisticsCounters.hpp>
 
 #include <string>
+#include <ctime>
 #include <unordered_map>
 #include <vector>
 
